js/src/product-feed/issues-table-card/issues-table-data-modal.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import GridiconExternal from 'gridicons/dist/external';
+
+/**
+ * Internal dependencies
+ */
+import AppModal from '.~/components/app-modal';
+import AppButton from '.~/components/app-button';
+
+/**
+ * Renders a modal showing information about a Google Merchant Center issue
+ *
+ * @fires gla_documentation_link_click with { context: 'issues-data-table-modal' }
+ * @param {Object} params Component params
+ * @param {Object} params.issue The issue to be rendered in the modal
+ * @param {Function} params.onRequestClose Onnclose callback function
+ */
+const IssuesTableDataModal = ( { issue, onRequestClose = () => {} } ) => {
+	if ( ! issue ) return null;
```

Comment:
You are right, I think is actually better tor remove that line that adds confusion to the code, since its actually impossible to had an empty issue or non issue as a param, as you mentioned. 

Since we are iterating over the issues we know for sure the issue exist, also since having the `issue.action` prop is a condition to show the component we now as well that prop is there. 

We can definitely check if the issue has `issue.code `and `issue.href.` But that would impact as well in previous stages, so I guess that would be a bit of over-engineering.